pages/_en-US/other/godmode9i.md

```diff
@@ -6,20 +6,20 @@ title: GodMode9i
 description: Information about GodMode9i
 ---
 
-[GodMode9i](https://github.com/DS-Homebrew/GodMode9i) is a file browser for the Nintendo DS ecosystem, taking inspiration from GodMode9 for the Nintendo 3DS. Dump cartridges, manage SD card files, browse NitroFS and more.
+[GodMode9i](https://github.com/DS-Homebrew/GodMode9i/) is an open source file browser for the Nintendo DS ecosystem, taking inspiration from GodMode9 for the Nintendo 3DS.
+GodMode9i has the ability to dump game cartridges, manage SD card files, browse NitroFS, calculate the SHA1 hash of files, edit binary files with a hex editor, and more.
 
 ## Dumping functionality
 
-To dump your cartridges, select the drive in the drives list and follow the on-screen prompts.
-- Nintendo DS (Lite) consoles can dump Slot-2/Game Boy Advance cartridges if running from Slot-1, or Slot-1/Nintendo DS cartridges if running from Slot-2
-- On the Nintendo DSi family of systems, launch GodMode9i with Unlaunch installed to allow access to dump Slot-1
+To dump game cartridges, select the drive in the drives list and follow the on-screen prompts.
+- On the Nintendo DS and Nintendo DS Lite, GodMode9i can dump Slot-2/Game Boy Advance cartridges if running from Slot-1 flashcard, or Slot-1/Nintendo DS cartridges if GodMode9i is running from a Slot-2 flashcard
+- On the Nintendo DSi family of systems, Unlaunch is needed for access to Slot-1
    - Follow [dsi.cfw.guide](https://dsi.cfw.guide/) for instructions on how to install Unlaunch
 - On the Nintendo 3DS family of systems, launch GodMode9i from a CFW-environment
```

Comment:
If for DSi we use "Unlaunch is needed for this", then it should also apply here as "a CFW environment is needed to use GodMode9i"